src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './variables';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-family: $default-font;
+  font-size: 30px;
```

Comment:
It's better to avoid specifying font styles on the `body` selector. Consider creating a base class for your text styles.